### PR TITLE
New Feature added

### PR DESCRIPTION
**Pull request by nicsaal**
App Name Changed in manifest file from Basic Location Sample > Where in the world is ...

### CODE REVIEW

File: BasicLocationSample/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@ limitations under the License.
             android:value="@integer/google_play_services_version" />
         <activity
             android:name=".MainActivity"
-            android:label="@string/app_name" >
+            android:label="Where in the World is ..." >
```
Comment (oliversaal) [inline BasicLocationSample/app/src/main/AndroidManifest.xml]: This is actually not the right place to change the name. The "@string/app_name" is a reference to the strings.xml file which stores all these values. Check out app/src/main/res/values/strings.xml. To address, revert this change to AndroidManifest.xml and apply the change to strings.xml. Make a new "Code review" commit and push that change for review.

**Comment by oliversaal** [comment]
@nicsaal , I guess in future we'll probably titled new pull requests something other than "New Feature"

**Comment by nicsaal** [comment]
So I tried following your instructions. Let me know if it looks better and whether I can merge with master.
